src/objects/Plan_Requirement__c.object

```diff
@@ -156,9 +155,9 @@
     </fields>
     <fields>
         <fullName>Sequence__c</fullName>
-        <description>The progression of courses</description>
+        <description>This requirement's place in the Program Plan, for example, "1" indicates the first semester. Use this field to indicate the overall progression for the entire Program Plan, or to sort the order of sub-requirements within a single parent Plan Requirement.</description>
```

Comment:
Do we need to consider double quote escaping here as well?

Reply:
we haven't been that I am aware of.